src/EditorFeatures/CSharp/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -36,7 +36,9 @@ internal class CSharpNavigationBarItemService : AbstractNavigationBarItemService
                                   SymbolDisplayParameterOptions.IncludeName |
                                   SymbolDisplayParameterOptions.IncludeDefaultValue |
                                   SymbolDisplayParameterOptions.IncludeParamsRefOut,
-                miscellaneousOptions: SymbolDisplayMiscellaneousOptions.UseSpecialTypes | SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral);
+                miscellaneousOptions: SymbolDisplayMiscellaneousOptions.UseSpecialTypes |
+                                      SymbolDisplayMiscellaneousOptions.AllowDefaultLiteral |
+                                      SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier);
```

Comment:
dumb question.  why is this an option?  when do you *not* want this?

Reply:
The compiler was choosing for some error messages to not show nullability if nullability didn't matter just to keep error messages shorter, so the option exists. The default symbol display does opt this in, but stuff where we chose a custom format we get to specify all the flags manually.